<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_dashboard">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clients">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Clients</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_diet_plans">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Diet Plans</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_settings">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_dashboard">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>380</y>
          <width>201</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Dashboard</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_clients">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_delete">
            <property name="text">
             <string>Delete Client</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_add_new">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Add New Client</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_client_detail">
       <widget class="QPushButton" name="btn_back_to_list">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_client_name">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>291</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS UI Gothic</family>
          <pointsize>24</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Client Detail</string>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>651</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>90</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Phone Numer:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_client_phone">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>90</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>....</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Notes:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lbl_client_notes">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>120</y>
          <width>441</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>....</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_edit">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>10</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Edit Client</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_add_measurement">
        <property name="geometry">
         <rect>
          <x>382</x>
          <y>110</y>
          <width>221</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Add Measurement</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="table_measurements">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>511</width>
          <height>391</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Weight (kg)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Body Fat (%)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Muscle (kg)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Metabolic Age</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Notes</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="page_add_client">
       <widget class="QPushButton" name="btn_save">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>120</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_cancel">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>130</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>cancel</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txt_name">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>240</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>enter name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="txt_phone">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>240</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>555 123 45 67</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="txt_notes">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>350</y>
          <width>104</width>
          <height>87</height>
         </rect>
        </property>
        <property name="plainText">
         <string>notes..</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_diet_plans">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>410</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Diet Plans</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_settings">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>350</y>
          <width>151</width>
          <height>101</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
